### PR TITLE
Rmasuti add caption

### PR DESCRIPTION
**Pull request by rajeshmasuti**
# Caption

### CODE REVIEW

**Review by vtiwari27 — APPROVED** [review]
Nice Caption

File: _posts/0000-01-04-rajeshmasuti.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![steve-harvey](https://cloud.githubusercontent.com/assets/16547949/25401187/0af0e288-29c3-11e7-9a88-38efd8b375cc.jpg)
 {: .center}
```
Comment (dmudigonda) [inline _posts/0000-01-04-rajeshmasuti.md:8]: Looks good!

File: _posts/0000-01-04-rajeshmasuti.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
...
 
-Caption here
```
Comment (dmudigonda) [inline _posts/0000-01-04-rajeshmasuti.md:9]: thought this reviewing is important

File: _posts/0000-01-04-rajeshmasuti.md [hunk 3]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![steve-harvey](https://cloud.githubusercontent.com/assets/16547949/25401187/0af0e288-29c3-11e7-9a88-38efd8b375cc.jpg)
```
Comment (dmudigonda) [inline _posts/0000-01-04-rajeshmasuti.md:6]: I don't really think so

**Review by dmudigonda — APPROVED** [review]

File: _posts/0000-01-04-rajeshmasuti.md (hunk 3 above)
Comment (rajeshmasuti) [reply _posts/0000-01-04-rajeshmasuti.md:6]: Steve has a habit of making the wrong announcement. :smiley:

**Review by dmudigonda — APPROVED** [review]

File: _posts/0000-01-04-rajeshmasuti.md (hunk 2 above)
Comment (vtiwari27) [reply _posts/0000-01-04-rajeshmasuti.md:9]: Looks Good!

**Review by vtiwari27 — APPROVED** [review]

**Pull request merged**